content/sql/concepts/commands/terms/top/top.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'TOP'
+Description: 'Returns a specified number of rows from the top of the result.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'SQL Server'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Returns a specified number of rows from the top of the result.
```

Comment:
```suggestion
The **`TOP`** command returns a specified number of rows from the top of the result.
```

The opening line of the entry should begin with the concept/term at hand, and the first use of it should be **bolded** as suggested above.